### PR TITLE
Housekeeping: improve docs side and top nav

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes no issue but opens up for adding a 'Localization' page to the side nav in #3694 

## What is the new behavior?
Previously, the top navigation that appear on large screens was collapsed into the side-menu on small screens. 
Now we always show links that were previously in the top navigation in the side nav on all screen sizes.

On small screens the open/close button for the side navigation is moved to the left side of the page to where the side nav is, and the logo is moved to the centre. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts [hunk 1]
```diff
@@ -14,32 +16,58 @@ import { filter } from 'rxjs/operators';
...
+  sideNavGroups: { [key: string]: QueryList<ElementRef<HTMLAnchorElement>> };
 
   constructor(private router: Router) {}
 
-  navigationLinks = navigationItems;
+  ngAfterViewInit(): void {
+    this.sideNavGroups = {
+      resources: this.resourceLinks,
+      components: this.componentLinks,
+    };
```
Comment (jakobe) [inline apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Any reason for this not being initialized in `OnInit`? Or even in a field initializer...

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts [hunk 2]
```diff
@@ -14,32 +16,58 @@ import { filter } from 'rxjs/operators';
...
+    {
+      name: 'Extensions',
+      path: '/home/extensions',
+    },
+    {
+      name: 'Changelog',
+      externalUrl: 'https://github.com/kirbydesign/designsystem/releases',
+    },
+    { name: 'GitHub', externalUrl: 'https://github.com/kirbydesign/designsystem' },
+  ];
```
Comment (jakobe) [inline apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Consider extracting to separate file as we do with app routes...

**Review by jakobe — COMMENTED** [review]
Really nice improvements 👌🏻 ❤️

**Comment by Fuzzy3** [comment]
This is an AWESOME PR !! Nicely done ! 🕺

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts (hunk 1 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: I needed the ViewChildren queries to resolve first with the original approach, but made it a getter as you suggested 👏 508df3284

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts (hunk 2 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Good point, changed it so we get the resource links from the app routes instead, similar to how we get showcase links: 094cee314

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts [hunk 3]
```diff
@@ -87,9 +95,11 @@ export class SideNavComponent implements OnInit {
     }
   }
 
-  onLinksArrowUpDown(event: KeyboardEvent) {
+  onLinksArrowUpDown(event: KeyboardEvent, sideNavGroupKey: string) {
     event.preventDefault();
-    const listElements: HTMLAnchorElement[] = this.componentLinks.map((link) => link.nativeElement);
+    const listElements: HTMLAnchorElement[] = this.sideNavGroups[sideNavGroupKey].map(
```
Comment (jakobe) [inline apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Maybe inline and simpify?
```suggestion
    const listElements: HTMLAnchorElement[] = (sideNavGroupKey === 'resources' ? this.resourceLinks : componentLinks).map(
```

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts [hunk 4]
```diff
@@ -69,12 +73,16 @@ export class SideNavComponent implements OnInit {
     this.applyComponentFilter('');
   }
 
+  private mapResourcesRoutes() {
+    this.filteredResourceRoutes = this.router.config
```
Comment (jakobe) [inline apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Could we import the routes same way as showcase routes?
```suggestion
    this.filteredResourceRoutes = appRoutes
```

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts [hunk 5]
```diff
@@ -14,14 +15,12 @@ import { filter } from 'rxjs/operators';
 import { kebabToTitleCase } from '@kirbydesign/designsystem';
 
 import { routes } from '../../showcase/showcase.routes';
```
Comment (jakobe) [inline apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Could we import app routes for resources same way as showcase routes?
```suggestion
import { appRoutes } from '../../app.routes';
import { showCaseRoutes } from '../../showcase/showcase.routes';
```

File: apps/cookbook/src/app/page/side-nav/side-nav.component.ts (hunk 3 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/page/side-nav/side-nav.component.ts]: Yeah, I was designing a tiny bit for the future here, but fine by me to remove it!

**Review by jakobe — APPROVED** [review]
:shipit:

**Pull request merged**